app/models/translation_override.rb

```diff
@@ -96,6 +98,10 @@ def check_interpolation_keys
     end
   end
 
+  def transform_pluralized_key(key)
+    match = key.match(/(.*)\.(zero|two|few|many)$/)
+    match ? match.to_a.second + '.one' : key
```

Comment:
This should use the `other` key. Usually it doesn't matter, but `other` is the correct key to use.  Sorry for nitpicking. 😉